static/app/views/releases/list/index.tsx

```diff
@@ -343,9 +344,87 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAdoptionChart(activeDisplay: DisplayOption) {
+    const {location, selection, organization} = this.props;
+    const {hasSessions, releases, reloading} = this.state;
+
+    const selectedProjectId =
+      selection.projects && selection.projects.length === 1 && selection.projects[0];
+    const selectedProject = organization.projects?.find(
+      p => p.id === `${selectedProjectId}`
+    );
+
+    if (
+      this.shouldShowLoadingIndicator() ||
+      !releases?.length ||
+      !selectedProject ||
+      !hasSessions
+    ) {
+      return null;
+    }
+
+    return (
+      <ReleaseHealthRequest
```

Comment:
This will need an adjustment so that it does not load additional 3 requests that we do not need.